Lzhl_tcp.cpp

```diff
@@ -54,7 +54,11 @@ SOCKET lzhl_socket( int af, int type, int protocol )
 
 SOCKET lzhl_accept( SOCKET s, struct sockaddr* addr, int* addrlen )
```

Comment:
I suggest add a note above this function that we have added the non-windows stuff

`// TheSuperHackers @feature vitimiti 13/03/2024 Your text here`